autoload/neomake/makers/ft/php.vim

```diff
@@ -27,9 +27,8 @@ endfunction
 function! neomake#makers#ft#php#phpcs() abort
     let l:args = ['--report=csv']
 
-    "Add standard argument if one is set.
-    if exists('g:neomake_php_phpcs_args_standard')
-        call add(l:args, '--standard=' . expand(g:neomake_php_phpcs_args_standard))
+    if exists('g:neomake_php_phpcs_args_post')
+        call add(l:args, , g:neomake_php_phpmd_args_post))
```

Comment:
> ./autoload/neomake/makers/ft/php.vim:31:26:Error: EVP_0: unexpected token: ,

Should be `call add(l:args, g:neomake_php_phpmd_args_post)`

But then it should also take into account buffer-local settings.
This could be done with `neomake#utils#GetSetting`, i.e. `call extend(l:args, neomake#utils#GetSetting('args_post', {}, [], ['php'], bufnr('%')))`.

Similar below.